pkgs/servers/home-assistant/parse-requirements.py

```diff
@@ -64,9 +63,11 @@ def name_to_attr_path(req):
     names = [req]
     # E.g. python-mpd2 is actually called python3.6-mpd2
     # instead of python-3.6-python-mpd2 inside Nixpkgs
-    if req.startswith('python-'):
+    if req.startswith('python-') or req.startswith('python_'):
         names.append(req[len('python-'):])
     for name in names:
+        # treat "-" and "_" equally
+        name = re.sub('[-_]', '[-_]', name)
```

Comment:
replacement is a regex as well?

Reply:
Yeah, because I need to replace both, - and _ at the same time. I don't think `string.replace()` can do that.

Reply:
I could do
```py
name = name.replace('_', '-')
name = name.replace('-', '[-_]')
```